LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/intrinsic-sizing/masonry-intrinsic-sizing-rows-003-mix2-expected.html

```diff
@@ -38,22 +38,22 @@
 <body>
 
 <section>
-<grid>
```

Comment:
Why does the first column here have to be a fixed value? Why is it simply not based off the size of the content?

Reply:
If you look at row 3 you will notice that the height is 1.4ch, which will cause a longer phrase like “3 3” to have a larger width in this first “column”. Since we are in a Grid here the most straightforward way to achieve the look we wanted was for the first column to be a fixed size. The “5 5” needs to line up right next to the “1” not extend far out.

Reply:
Does it also work if you just remove the `grid-column: span 2` from the 5 5 so that it only spans one column? If that works that might be better imo since it introduces fewer "magic numbers"

Reply:
Yes, that does work.